src/ocean/Account.ts

```diff
@@ -104,18 +104,47 @@ export default class Account extends Instantiable {
     ] as AbiItem[]
 
     let result = null
-
+    const decimals = await this.getTokenDecimals(TokenAdress)
     try {
       const token = new this.web3.eth.Contract(minABI, TokenAdress, {
         from: this.id
       })
-      result = this.web3.utils.fromWei(await token.methods.balanceOf(this.id).call())
+      const balance = await token.methods.balanceOf(this.id).call()
+      result = balance.div(10 ** decimals).toString()
```

Comment:
`balance.div(` is not working here.
I tried replacing with `result = balance / 10 ** decimals` and it worked perfectly.